sormas-rest/src/main/java/de/symeda/sormas/rest/DashboardResource.java

```diff
@@ -76,6 +77,25 @@ public DashboardCaseStatisticDto getDashboardCaseStatistic(@RequestBody Dashboar
 		return FacadeProvider.getDashboardFacade().getEpiCurveSeriesElementsPerPresentCondition(dashboardCriteria);
 	}
 
+	@POST
+	@Path("/epiCurveContactStatusClassification")
+	public Map<Date, Map<ContactClassification, Long>> getEpidemiologicalCurveContactClassification(
```

Comment:
I'd suggest to call these methods exactly like the facade methods, makes it clearer.